lib/gears/color.lua

```diff
@@ -59,7 +59,7 @@ local pattern_cache
 -- Max 4 chars per channel.
 --
 -- @param col The color to parse
--- @treturn table 4 values representing color in RGBA format (each of them in
+-- @return 4 values representing color in RGBA format (each of them in
```

Comment:
ldoc supports to use multiple `@return`/`@treturn` tags. Please document all 4 returned value with their respective types.

Reply:
I wasn't able to revert the second commit, or move it to a separate PR. Please feel free to close this PR. I will create two separate new PRs for the edits, considering also your comment above regarding return tags in ldoc.